packages/ayzek-api-telegram/index.ts

```diff
@@ -202,6 +201,9 @@ export class TelegramApi extends Api {
 	}
 	async processMessageUpdate(messageUpdate: any) {
 		const message = await this.parseMessage(messageUpdate);
+		if (message.text.startsWith('/') && !message.text.startsWith('//') && message.text.length != 1) {
```

Comment:
Telegram requires more complex logic with command parsing, you should check username embedded in command, and then strip it